lib/geokit/geocoders/google.rb

```diff
@@ -86,11 +86,12 @@ def self.construct_bias_string_from_options(bias)
 
       def self.parse_json(results)
         case results["status"]
-        when "OVER_QUERY_LIMIT" then raise Geokit::Geocoders::TooManyQueriesError
+        when "OVER_QUERY_LIMIT" then raise Geokit::Geocoders::TooManyQueriesError, results['error_message']
+        when "REQUEST_DENIED" then raise Geokit::Geocoders::AccessDeniedError, results['error_message']
```

Comment:
Line is too long. [103/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.